src/napari_imagej/types/widget_mappings.py

```diff
@@ -11,7 +11,12 @@
 from napari.layers import Image
 
 from napari_imagej.java import jc
-from napari_imagej.widgets.parameter_widgets import numeric_type_widget_for
+from napari_imagej.widgets.parameter_widgets import (
+    DirectoryWidget,
+    OpenFileWidget,
+    SaveFileWidget,
+    numeric_type_widget_for,
```

Comment:
Maybe we could replace the direct imports on the file widget classes with something like `numeric_type_widget_for`?